iroha/src/kura.rs

```diff
@@ -404,4 +407,46 @@ mod tests {
             .await
             .expect("Failed to store block into Kura.");
     }
+
+    #[tokio::test]
+    #[ignore = "Should be run seprately to print block size"]
+    async fn block_size_metric() {
+        println!("File size of a block with 1 transaction with 1 Transfer instruction is:",);
+        for max_faults in 0i32..5i32 {
+            let n_validators = 3 * max_faults + 1;
+            let dir = tempfile::tempdir().unwrap();
+            let alice_xor_id =
+                <Asset as Identifiable>::Id::from_names("xor", "test", "alice", "test");
+            let bob_xor_id = <Asset as Identifiable>::Id::from_names("xor", "test", "bob", "test");
+            let transfer = Instruction::Transfer(TransferBox {
+                source_id: IdBox::AssetId(alice_xor_id).into(),
+                object: Value::U32(10).into(),
+                destination_id: IdBox::AssetId(bob_xor_id).into(),
+            });
+            let keypair = KeyPair::generate().expect("Failed to generate KeyPair.");
+            let tx = Transaction::new(
+                vec![transfer],
+                <Account as Identifiable>::Id::new("alice", "wonderland"),
+                1000,
+            )
+            .sign(&keypair)
+            .expect("Failed to sign.");
+            let tx = VersionedAcceptedTransaction::from_transaction(tx, 4096)
+                .expect("Failed to accept Transaction.");
+            let mut block = PendingBlock::new(vec![tx])
+                .chain_first()
+                .validate(&WorldStateView::new(World::new()), &AllowAll.into());
+            for _ in 0..n_validators {
+                block = block
+                    .sign(&KeyPair::generate().expect("Failed to generate KeyPair."))
+                    .unwrap();
+            }
+            let block = block.commit();
+            let block_store = BlockStore::new(dir.path()).unwrap();
+            block_store.write(&block).await.unwrap();
+            let metadata = fs::metadata(block_store.get_block_path(1)).await.unwrap();
+            let file_size = Byte::from_bytes(metadata.len() as u128).get_appropriate_unit(false);
+            println!("For {} validators: {}", n_validators, file_size);
+        }
```

Comment:
Also may be you can move loop body into some function?

Reply:
I don't think it makes sense for this simple test. Nothing is shared with the other tests.

Reply:
Yeah, but it would give more context and will be more readable